pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -367,7 +391,8 @@ func (c *Cluster) waitForMonsToJoin(mons []*monConfig) error {
 	}
 
 	// wait for the monitors to join quorum
-	err := waitForQuorumWithMons(c.context, c.clusterInfo.Name, starting)
+	sleepTime := 5
```

Comment:
Is that enough time?
Is the Mon Pod already expected to be running by then or could the Pod still need time to pull the image and/or start up on the node (e.g., busy cluster)?

Reply:
This isn't changing, the code just moved so i could unit test. There's a retry loop of 20 times, so it will retry for up to 4 minutes. If all these tests fail, the operator will start this whole thing and try again. But we should really make it smarter and wait longer if the pod is still in containerCreating stage.

Reply:
I see, 4 minutes should be enough.